app/serializers/answer_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class AnswerSerializer < ActiveModel::Serializer
+  attributes :id, :response, :user_id, :accepted, :cached_votes_up, :cached_votes_down
```

Comment:
Totes! At the first pass I was following [this tutorial](http://vaidehijoshi.github.io/blog/2015/06/23/to-serialize-or-not-to-serialize-activemodel-serializers/) to a t.

I'm thinking I'll create a users serializer and build out the same private method across questions, answers, and  comments, and we can just send the same bundle for each. It's a small amount of data anyway.